### PR TITLE
Added 'Atlassian Git Cheatsheet'

### PR DESCRIPTION
**Pull request by keerthanasathya**
## What does this PR do?
I've added link to Git Cheatsheet

### Why is this valuable (or not)?
It is valuable because, it is beginner-friendly

### How do we know it's really free?
It is free

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by keerthanasathya** [comment]
#7190

File: books/free-programming-books-ar.md [hunk 1]
```diff
@@ -155,4 +155,5 @@
 
 * [مدخل إلى تجربة المستخدم](https://academy.hsoub.com/files/11-مدخل-إلى-تجربة-المستخدم-user-experience-ux/)
 
+
```
Comment (davorpa) [inline books/free-programming-books-ar.md:158]: Reverting superfluous change

```suggestion
```

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -116,12 +116,14 @@
...
     * [Git Cheat Sheet](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet.pdf) (PDF)
     * [Git Cheat Sheet - extended](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet-v2.zip) (PDF)
+* [Git Atlassian Cheat Seet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet) -Atlassian (Markdown)
 * [Git Cheat Sheet](https://programmingwithmosh.com/wp-content/uploads/2020/09/Git-Cheat-Sheet.pdf) - Moshfegh Hamedani (PDF)
 * [Git ściąga (pl)](https://training.github.com/downloads/pl/github-git-cheat-sheet/) - GitHub
 * [GitHub Actions Security Best Practices \[cheat sheet included\]](https://blog.gitguardian.com/github-actions-security-cheat-sheet/) - Thomas Segura, GitGuardian, C.J. May (online, PDF)
 * [GitHub Cheat Sheet](https://github.com/tiimgreen/github-cheat-sheet) - Tim Green (Markdown)
 
 
+
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:126]: Reverting superfluous change

```suggestion
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: more/free-programming-cheatsheets.md [hunk 3]
```diff
@@ -116,6 +116,7 @@
 * [Git Cheat Sheet](http://git.jk.gs) - Jan Krüger (PDF, SVG)
     * [Git Cheat Sheet](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet.pdf) (PDF)
     * [Git Cheat Sheet - extended](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet-v2.zip) (PDF)
+* [Git Atlassian Cheat Seet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet) -Atlassian (Markdown)
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:119]: Fixing resources info

```suggestion
* [Git cheat sheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet) - Atlassian
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: more/free-programming-cheatsheets.md [hunk 4]
```diff
@@ -116,6 +116,7 @@
 * [Git Cheat Sheet](http://git.jk.gs) - Jan Krüger (PDF, SVG)
     * [Git Cheat Sheet](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet.pdf) (PDF)
     * [Git Cheat Sheet - extended](https://jan-krueger.net/wordpress/wp-content/uploads/2007/09/git-cheat-sheet-v2.zip) (PDF)
+* [Git cheat sheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet) - Atlassian
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:119]: Adding PDF format note

```suggestion
* [Git cheat sheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet) - Atlassian (PDF)
```

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**